errbot/backends/slack.py

```diff
@@ -555,11 +555,15 @@ def send_card(self, card: Card):
         if isinstance(card.to, RoomOccupant):
             card.to = card.to.room
         to_humanreadable, to_channel_id = self._prepare_message(card)
-        attachment = {}
+        attachment = {
+            'mrkdwn_in': [],
+        }
         if card.summary:
             attachment['pretext'] = card.summary
+            attachment['mrkdwn_in'].append('pretext')
```

Comment:
It is not, what Slack calls "markdown" is something weird. The markdown we use on Errbot is defined by the original markdown syntax: http://daringfireball.net/projects/markdown/syntax + some extra from https://pythonhosted.org/Markdown/extensions/extra.html like table and colors.
Whatever the backends defines (ansi like console, IRC colors, xhtml for xmpp, other gitter markdown flavor etc...) we need to convert the errbot format to it do plugins can write in a consistent markdown. So accepting here the weird slack format straight from the plugins will brake ... hipchat for one.